modules/ximgproc/include/opencv2/ximgproc/slic.hpp

```diff
@@ -138,21 +140,22 @@ class CV_EXPORTS_W SuperpixelSLIC : public Algorithm
 
```

Comment:
Brief description of this function should be changed to something like `Initializes a SuperpixelSLIC object.` It's an old copy-paste error.